Client/Ecosia/L10N/String.swift

```diff
@@ -161,7 +161,7 @@ extension String {
         case after = "After"
         case treesPlantedByTheCommunity = "trees planted by the Ecosia community"
         case treesPlantedByTheCommunityCapitalized = "Trees planted by the Ecosia community"
-        case investedIntoClimateAction = "invested into climate action"
+        case dedicatedToClimateAction = "dedicted to climate action"
```

Comment:
We just need to make sure we add the translations ourselves in Transifex after the source file is updated. This is because we already have them and do not need translator to do it.

Have you considered this step to be done after the merge?

Reply:
💯.
In fact will plan to do exactly that, given Transifex suggests the translations with the match percentage 👌.
The changes were done purely to keep the flow consistent